Azure/azure-cli
# [App Service] BREAKING CHANGE: Redact output on deployment source config

**Related command**
`az functionapp deployment source config`
`az webapp deployment source config`

**Description**
Currently, these two commands return a secret as part of the output. In this PR, we are redacting the secret to prevent accidental security leaks through CI logs.

**Testing Guide**
1. Create functionapp (`az functionapp create`)
2. Configure deployment (`az functionapp deployment source config`)

Reference document for object returned by the deployment source config commands: https://learn.microsoft.com/en-us/rest/api/appservice/web-apps/get-source-control#sitesourcecontrol

---

This checklist is used to make sure that common guidelines for a pull request are followed.
